pkg/apis/core/validation/validation.go

```diff
@@ -4356,10 +4357,24 @@ func ValidateConfigMap(cfg *core.ConfigMap) field.ErrorList {
 		for _, msg := range validation.IsConfigMapKey(key) {
 			allErrs = append(allErrs, field.Invalid(field.NewPath("data").Key(key), key, msg))
 		}
+		// check if the value stored consists entirely of valid UTF-8-encoded runes
+		if !utf8.ValidString(cfg.Data[key]) {
+			allErrs = append(allErrs, field.Invalid(field.NewPath("data").Key(key), key, "invalid byte sequence in UTF-8"))
```

Comment:
This is strictly a breaking change.  Do we really need this validation?  If something violated this before, and it worked, do we care?

Reply:
I think you're right, as painful as it is to let people continue shooting themselves in the foot

Reply:
ok removing it now